ServiceTier/Services/LogInService.cs

```diff
@@ -61,7 +62,8 @@ private static string GenerateToken(User user)
 
             var tokenString = new JwtSecurityTokenHandler().WriteToken(token);
 
-            return tokenString;
+            return JsonConvert.SerializeObject(new
+            { access_token = tokenString });
```

Comment:
make it one line

Reply:
@roman-fay done

Reply:
so this is commit into develop, but the change was already committed into release branch?

Reply:
yes without it sign in doesn't work properly